client/src/GuessGame/DataMap.jsx

```diff
@@ -0,0 +1,112 @@
+import PropTypes from 'prop-types';
```

Comment:
Great job at implementing this game, I don't fully understand what is happening in this file but its cool to see all the logic and functionality needed to execute this game.